### DISCUSSION TITLE
Reason for this commit: Fix restart Info JSON messages

### DISCUSSION THREAD

**Discussion by ortegafernando** [arendst/Tasmota#12084] [Q&A] [answered]
Hi, for my better understanding, which is the reason for this commit?

https://github.com/arendst/Tasmota/commit/e94b997627e97162a84a0bef95c25b9b192f7413#diff-61a9217119fcc591ba23c2490190b967ed63296e6339e26bac8e70b85b7f2ce6

I had already configured NODERED flows and now, with new realase I need to change them because of this commit. I also will have to update all devices to the new JSON messages.

I dont understand what is the reason of this change. I am sure that it is beneficial, I don't discuss that, I just want to know the reason to try o understand the philosophy of the firmware

Thanks.

**Comment by arendst** [comment, marked as answer]
To support rules better by discriminating subfields

**Comment by ortegafernando** [comment]
Thanks a lot. May be I did wrong, but I tried to search it in changelog but I couldn't find it. In future revisions of MQTT payloads, if you find it usefull, please consider to add it to changelog. For me, it is.
Thanks a lot for your effort, please consider this as an advice, not as a critical.
Thakns.
